Packs/PrismaCloudCompute/ReleaseNotes/1_3_2.md

```diff
@@ -0,0 +1,49 @@
+
+#### Scripts
+
+##### New: CreatePrismaCloudComputeComplianceReportButton
+
+- Generate a compliance report via button.
+
+##### New: CreatePrismaCloudComputeLink
+
+- This script creates a link back to the Prisma Cloud Compute instance.
+
+##### New: TagIndicatorButton
+
+- This is a wrapper around the setIndicators script
+
+
+#### Layouts
+
+##### New: Prisma Cloud Compute Audit Layout v2
+
+#### Incident Fields
+
+- New: **Prisma Cloud Compute Defender Logs**
+
+- New: **Prisma Audit Table**
+
+- **Prisma Cloud Compute Category**
```

Comment:
Is this a new incident field?  An updated incident field?

Reply:
added in error.  removed now.